ccmlib/scylla_node.py

```diff
@@ -1272,20 +1277,74 @@ def _import_executables(self, install_dir):
             self.node.node_install_dir = self.orig_install_dir
             raise NodeUpgradeError("Failed to import executables files. %s" % exc)
 
-    def upgrade(self, upgrade_version: str):
+    def _recover_system_tables(self):
+        """
+        Part of the rollback procedure is to restore system tables.
+        For this goal a snapshot will be taken before downgrade and restore the system tables after downgrade
```

Comment:
"        For this goal a snapshot will be taken before downgrade and restore the system tables after downgrade"

needs to be: `For this goal a snapshot will be taken before upgrade and restore the system tables before rollback`, no?